Gruntfile.js

```diff
@@ -29,18 +29,18 @@ module.exports = function(grunt) {
         pkg: packageInfo,
 
         jshint: {
+            options: {
+                jshintrc: '.jshintrc'
+            },
             player : [
-                'src/js/**/*.js'
+                'src/js/{,*/}*.js'
             ],
             tests : [
                 'test/{,*/}*.js'
             ],
             grunt : [
                 'Gruntfile.js'
-            ],
-            options: {
-                jshintrc: '.jshintrc'
```

Comment:
Why remove this? I think it is needed : https://github.com/gruntjs/grunt-contrib-jshint